docs/SETUP.md

```diff
@@ -302,6 +319,16 @@ your connector.
 
 ## Restarting ilp-kit after a server reboot
 
+```
+# Using npm
+$ cd ~/ilp-kit
+$ rm -rf node_modules
+$ npm install; npm rebuild; npm start 
```

Comment:
Ah, there's a confusion here; the header should be 'Make ilp-kit restart automatically after a server reboot', because that's what `pm2 startup` does. You don't want to have to ssh into your server after each reboot.

As discussed in slack as well, the easiest way to keep ilp-kit running when you close your ssh session, is maybe using `screen`. Other options are [nohup/script.sh](https://github.com/interledgerjs/ilp-kit/blob/master/docs/SETUP.md#launch-ilp-kit), systemd, pm2.